### PR TITLE
Sigulsign python3

### PR DESCRIPTION
**Pull request by sergiomb2**
- Switch to  python3 copied from https://pagure.io/releng/blob/master/f/scripts/sigulsign_unsigned.py

    - Try to fix koji authentication

### CODE REVIEW

**Comment by sergiomb2** [comment]
JFYI koji code is here https://pagure.io/koji/blob/master/f/koji/__init__.py#_2544

File: sigulsign_unsigned.py [hunk 1]
```diff
@@ -89,7 +92,7 @@ def __init__(self, arch=None):
             self.kojihub = "koji"
         self.koji_module = koji.get_profile_module(self.kojihub)
         self.kojisession = self.koji_module.ClientSession(self.koji_module.config.server, {'krb_rdns': False})
-        self.kojisession.ssl_login(CLIENTCERT, CLIENTCA, SERVERCA)
+        self.kojisession.ssl_login(CLIENTCERT, None, None)
```
Comment (kwizart) [inline sigulsign_unsigned.py]: Now that the serverca and clientca are uneeded, you can use self.kojisession.ssl_login(CLIENTCERT) instead.

File: sigulsign_unsigned.py [hunk 2]
```diff
@@ -240,14 +243,14 @@ def __init__(self, key, password=None, config_file=None, arch=None,
...
-            except:
-                fas_username = getpass.getuser()
+#            try:
+#                krb_realm = "RPMFUSION.ORG"
+#                if use_staging:
+#                    krb_realm = "STAGING.RPMFUSION.ORG"
+#
+#                fas_username = cccolutils.get_user_for_realm(krb_realm)
+#            except:
+            fas_username = getpass.getuser()
```
Comment (kwizart) [inline sigulsign_unsigned.py:254]: You can keep this part if (once) we will use kerberos. It shouldn't cause hard issue...

**Review by kwizart — CHANGES_REQUESTED** [review]
The only concern with theses changes is that we need to keep an older (python2) sigul working interfacing with koji (with python2)
If this script can be converted to python3, we need to test it under el7, but then we need to have both python2/3 koji bindings where el7 only has python2-koji.

So this is a little more complicated. maybe fedora31 has both bindings, but for now I would say el7 is more maintainable until we can migrate to el8 stack (with full python3). But that won't be this year...

**Pull request closed without merging**